nixos/lib/qemu-common.nix

```diff
@@ -22,7 +22,7 @@ rec {
         else throw "Unknown QEMU serial device for system '${pkgs.stdenv.hostPlatform.system}'";
 
   qemuBinary = qemuPkg: {
-    x86_64-linux = "${qemuPkg}/bin/qemu-kvm -cpu max";
+    x86_64-linux = "${qemuPkg}/bin/qemu-kvm -cpu qemu";
```

Comment:
I assume this is the same as `qemu64`?  If so, it's the default, so we could just omit the `-cpu`.

Reply:
I changed it to `qemu64`, I did not want to omit it. 3baacbf30f6b81af2b5bd3341da157093c0ba299